oas-generator/oas-generator-core/src/main/java/org/apache/servicecomb/toolkit/generator/util/ModelConverter.java

```diff
@@ -143,4 +176,27 @@ public static ObjectMapper mapper() {
 
     return mapper;
   }
+
+  public static Map<String, Type> getBeanProperties(Class cls) {
+    if (cls.isPrimitive()) {
+      return null;
+    }
+    Method[] declaredMethods = cls.getDeclaredMethods();
+    Map<String, Type> beanProperties = new HashMap<>();
+
+    for (Method method : declaredMethods) {
+      if (method.getName().startsWith("get")) {
+        if (method.getReturnType() != null && method.getReturnType() != void.class) {
+          String propName = method.getName().substring(3);
+          try {
+            cls.getDeclaredMethod("set" + propName, method.getReturnType());
+            beanProperties.put(method.getName(), method.getGenericReturnType());
+          } catch (NoSuchMethodException e) {
+            continue;
+          }
+        }
+      }
+    }
```

Comment:
For request payload, bean property could be a non-static public field, or a setter method. And the setter method:

1. is public, non-static
2. name should be `setAbc` or `setABC`, `setabc` is not a setter method
3. return value is void
4. only one parameter
5. no corresponding field is OK
6. no getter is OK

And for response body, bean property could be a non-static public field, or a getter method, And the getter method:

1. is public, non-static
2. name should be `getAbc`, `getABC`, `getabc` is not a getter method
3. return value is not void
4. no parameters
5. no corresponding field is OK
6. No setter is OK